app/src/main/java/me/thuanc177/kotsune/libs/animeProvider/allanime/AllAnimeTypes.kt

```
@@ -10,6 +10,73 @@ data class AllAnimePageInfo(
+            val datePart = parts[0].split("-")
+            val timePart = parts[1].split(":")
+
+            val year = datePart[0]
+            val month = getMonthName(datePart[1].toInt())
+            val day = datePart[2]
+            val hour = timePart[0].toInt()
+            val minute = timePart[1]
+
+            // Format: "September 29, 2023 at 1:11 PM"
+            "$month $day, $year at ${formatHour(hour)}:$minute ${if (hour < 12) "AM" else "PM"}"
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)